docs/farming.md

```diff
@@ -226,19 +239,31 @@ Does not matter if the node/farmer executable is the previous one or from the ne
 The reason we require this is, with every snapshot change, the network might get partitioned, and you may be on a different genesis than the current one.
 In plain English, these commands are like a `reset` button for snapshot changes.
 
-## Glossary for useful farmer commands
-
-Structure -> `subspace-farmer-x86_64-*-snapshot <COMMAND>`
-
-- `farm --reward-address WALLET_ADDRESS` : starts background plotting and farming together, farmed testnet coins will be sent to `WALLET_ADDRESS`
-- `farm` : starts background plotting and farming together, rewards are sent to auto-generated wallet (see `identity` commands below)
-- `wipe` : erases the plot and identity (including plot, commitment, object mappings and identity files)
-- `identity import-from-mnemonic "spell out your seed phrase here"` : imports your existing identity from your seed phrase (not recommended! use `--reward-address` instead)
-- `identity view` : displays SS58 address (this is the same as `identity view --address`) where farmed testnet coins will be sent
-- `identity view --mnemonic` : displays mnemonic phrase of auto-generated wallet (sensitive information, keep this private, not very useful if `--reward-address` was used)
+## Farmer Commands
+
+Structure -> `./FARMER_FILE_NAME <COMMAND> <SUBCOMMAND>`
+
+- `erase-plot`: Erase existing plot (doesn't touch identity)
+  - `--custom-path <CUSTOM_PATH>`: Use custom path for data storage instead of platform-specific default.
+- `farm`: Start a farmer using previously created plot
+  - `--bootstrap-node <BOOTSTRAP_NODE>`: Multiaddrs of bootstrap node to connect to on startup, multiple are supported
+  - `--custom-path <CUSTOM_PATH>`: Custom path for data storage instead of platform-specific default
+  - `--listen-on <LISTEN_ON>`: Multiaddrs to listen on for subspace network, for instance '/ip4/0.0.0.0/tcp/0', multiple are supported, subspace networking is disabled when non is specified.
+  - `--node-rpc-url <NODE_RPC_URL>`: WebSocket RPC URL of the Subspace node to connect to (default: ws://127.0.0.1:9944)
+  - `--reward-address <REWARD_ADDRESS>`: Address for farming rewards
+  - `-w`, `--ws-server-listen-addr <WS_SERVER_LISTEN_ADDR>`: Host & Port where built-in WebSocket RPC server should listen for incoming connections (default: 127.0.0.1:9955)
+- `help`: Print this message or the help of the given subcommand(s)
+- `identity`: Identity management
```

Comment:
No need to expand on identity management, it was specifically reduced to bare minimum that is useful and should better be removed altogether by now. Importing/exporting of mnemonic is mostly and developer's tool right now not intended for end users and might be gone in the future altogether.